Practice2/src/styles/variables/main-var.css

```diff
@@ -0,0 +1,33 @@
+:root {
+  /* Color */
+  --c-light: #fff;
+  --c-dark: #4b4b4b;
+  --c-warning: #EBCB80;
```

Comment:
It's not consistent. Should be lowercase like other colour codes.

Reply:
Fixed